src/libraries/System.Private.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/HelperMarshal.cs

```diff
@@ -72,6 +72,16 @@ private static string StoreArgumentAndReturnLiteral(string s)
             return "1";
         }
 
+        private static string StoreAndReturnNew(string s)
+        {
+            var sb = new System.Text.StringBuilder();
+            sb.Append("Got:");
+            sb.Append(' ');
+            sb.Append(s);
+            _stringResource = sb.ToString();
+            return _stringResource;
+        }
```

Comment:
Is it important that this use StringBuilder?  Seems like it could just be:
```C#
private static string StoreAndReturnNew(string s) =>
    _stringResource = "Got: " + s;
```

Reply:
I explicitly wanted StringBuilder because I wanted to test different code path than StoreArgumentAndReturnLiteral() above.

Reply:
Unless s is an empty string,  using concat won't return a literal.  It's a test so not a big deal,  but this seems unnecessarily complex.